docqueries/bellman_ford/doc-pgr_edwardMoore.result

```diff
@@ -96,8 +96,8 @@ SELECT * FROM pgr_edwardMoore(
    5 |        3 |        11 |       5 |    5 |   -1 |    0 |        2
 (5 rows)
 
--- q7
--- q8
+/* -- q7 */
```

Comment:
Please manually add 2 blank lines
Should look like this.

/* -- q7 */


/* -- q8 */

Make sure that you are not introducing the windows end of line

Reply:
thank you!